app/src/androidTest/java/org/fossasia/susi/ai/login/LoginActivityTest.kt

```diff
@@ -59,18 +59,23 @@ class LoginActivityTest {
         Log.d(TAG, "running testUIViewsPresenceOnLoad..")
 
         // checks if email input field is present
+        onView(withId(R.id.email)).perform(scrollTo())
         onView(withId(R.id.email)).check(matches(isDisplayed()))
 
```

Comment:
why did you add this?

Reply:
@batbrain7 While trying to fix failing Circle CI tests, I made this change. Currently, I am going through tests, trying to write unit tests and also trying to fix these errors. So, I hope this will be fixed soon.